### DISCUSSION TITLE
NextJS13 Head example {params}?

### DISCUSSION THREAD

**Discussion by eric-mathison** [vercel/next.js#42163] [Help]
In the example for the head.tsx file in the documentation, the function accepts params. 
Maybe I'm missing this, but how can you pass arguments to this head function? 

```
app/post/[slug]/head.tsx

export default async function Head({ params }) {
  const post = await getPost(params.slug);

  return (
    <>
      <title>{post.title}</title>
    </>
  )
}
```

**Comment by dejurin** [comment]
It is obvious that this param is [slug];

`/post/foobar`

`{ slug: 'foobar' }`

Most likely that's all there is.
